plugins/flashrom/fu-flashrom-internal-device.c

```diff
@@ -117,6 +119,12 @@ fu_flashrom_internal_device_write_firmware(FuDevice *device,
 	if (blob_fw == NULL)
 		return FALSE;
 
+	/* progress */
+	fu_progress_set_id(progress, G_STRLOC);
+	fu_progress_add_flag(progress, FU_PROGRESS_FLAG_GUESSED);
+	fu_progress_add_step(progress, FWUPD_STATUS_DEVICE_WRITE, 90);
```

Comment:
flashrom doesn't have an erase step?

Reply:
It does, but it's hidden in `flashrom_image_write()` -- which is yet another frustration with the libflashrom API...